src/UIKit/UIAccessibility.cs

```diff
@@ -317,20 +317,22 @@ static void RequestGuidedAccessSessionBridge (bool didSucceed)
 			}
 		}
 
+		[iOS (10,0), TV (10,0)]
 		[DllImport (Constants.UIKitLibrary)]
 		static extern bool UIAccessibilityIsAssistiveTouchRunning ();
-		[iOS (10,0)]
+		[iOS (10,0), TV (10,0)]
 		public static bool IsAssistiveTouchRunning {
 			get {
 				return UIAccessibilityIsAssistiveTouchRunning ();
 			}
 		}
 
 #if !TVOS
+		[iOS (10,0), TV (10,0)]
```

Comment:
really ? because it's under `!TVOS` and I'm fairly sure strong pointed them as unavailable.